### DISCUSSION TITLE
How best to engage with Redgate after their PASS content acquisition?

### DISCUSSION THREAD

**Discussion by sqlatspeed** [dataplat/DataSaturdays#111] [Q&A]
What is the best way for this DataSaturdays collective to engage with Redgate following their acquisition of PASS content (and specifically the SQLSaturday domain and branding)?

**Comment by way0utwest** [comment]
Looking for some ideas here as well. We (Redgate) were thinking to set up some forum space somewhere and take debates/discussions/etc. SQLServerCentral might be the easiest spot, but certainly there might be a need to support a variety of platforms. 

For SQL Saturday, perhaps this is the best spot.  Not sure

**Reply by way0utwest** [reply]
We have a forum here for threads: https://www.sqlservercentral.com/forums/forum/sqlservercentral-com/the-future-of-community

**Comment by rabryst** [comment]
I don't want the community (or Redgate) to assume that PASS is the sole owner of the Microsoft data platform community (that should be a collective decision). The PASS brand has already lost a lot of goodwill in the last 12 months, and this purchase may dilute it further.

If anything, the independence of the Data Saturdays initiative should continue. Redgate owns a brand and historic content, but it's the people in our local communities (i.e. us) who arrange everything. If Redgate wants to help, I'm OK with that. At any rate we still need to produce all the things we've identified in the last 6 weeks.

**Reply by amtwo** [reply]
It is a very important detail that PASS has lost a lot of goodwill among speakers & organizers with respect to the SQLSaturday brand. SQLSaturday has strong name recognition with attendees of local events, but many of the volunteers who make those events happen have soured on the brand. 

Redgate certainly has the opportunity to fix that soured relationship with volunteers, but it is certainly something that Redgate needs to keep in mind as an obstacle that needs to be managed. Overcoming that obstacle is ultimately a combination of Product and Marketing efforts.

**Reply by way0utwest** [reply]
While I think PASS lost some goodwill, I know many organizers were happy and pleased with how the platform/franchise was managed.

That being said, we need to find a way to drive things forward in a more positive, inclusive, and exciting way.
